Scripts/ai-duty.sh

```
@@ -60,6 +62,92 @@ cleanup_simulators() {
+collect_notify_targets() {
+  local ringi approval
+  ringi=$(gh issue list -R hiroky1983/game_collection --label "ringi:pending" --state open \
+    --json number --jq '[.[].number] | map(tostring) | join(" ")' 2>/dev/null) || return 0
+  # 承認待ち = ai:proposed のうち会長のハンコがまだ無いもの。着手済み・外部イベント待ち（blocked）と、
+  # 上の決裁待ちに既に出ているものは重複するので除く
+  approval=$(gh issue list -R hiroky1983/game_collection --label "ai:proposed" --state open \
+    --json number,labels \
+    --jq '[.[] | ([.labels[].name]) as $l
+          | select(($l | index("ai:approved")) == null and ($l | index("ai:in-progress")) == null
+                   and ($l | index("blocked")) == null and ($l | index("ringi:pending")) == null)
+          | .number] | map(tostring) | join(" ")' 2>/dev/null) || return 0
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)